01-python/safety/hospital.py

```
@@ -1,3 +1,15 @@
-def draw_hospital():
-    print("hospital not found")
-    return
+def draw_hospital(width, height):
+    #width >= 4
+    #height >= 3
+    if width >= 4:
+        indent = width - 4
+        indent = indent // 2
+    else:
+        width = 4
```

modifying the parameters is not a good idea. If I use your function with parameter greater than 4 it may take me days to figure out why it's not printing something with width greater than 4... If you don't want to support that width please raise an error instead